server/handlers/listUserHandler.py

```
@@ -18,7 +18,13 @@ def listUserHandler():
     response = list()
 
     for user_blob in users_blob.values():
-        user = user_schema.from_bytes_packed(user_blob)
+        try:
+            user = user_schema.from_bytes_packed(user_blob)
+        except:
+            # Added because this happened to me, but the condition needs to be clearer
+            logging.warn(msg="Corrupted DB@'users' : skipping user")
```

remove this completly. we don't mask error. We following an always die principle.